crypto/entropy.go

```diff
@@ -0,0 +1,36 @@
+package crypto
```

Comment:
How do we feel about naming and function design? 

- `entropy.go` ?
- `GenerateEntropy()`
- `GenerateHexNonce()`
    - Should we just prescribe hexadecimal as the encoding (in which case `GenerateNonce()`)?

Reply:
I like `entropy.go` and that `GenerateEntropy` is encoding agnostic. I prefer `GenerateNonce()` because I don't anticipate us writing other `Generate<Encoding>Nonce` functions, so I don't think we need the distinction. Let's mention in a doc comment that it's hex encoded.

Reply:
Agreed, done